promscale/installation/docker.md

```diff
@@ -3,8 +3,7 @@ If you want to try out Promscale in a test environment before getting started,
 you can use a pre-built Docker container.
 
 <highlight type="important">
-Using Promscale from a Docker container is for local testing purposes only. Do
-not use this installation method in a production environment.
+Running Promscale in a Docker container is not suitable for production use-cases. 
```

Comment:
Check the page now: https://docs.timescale.com/promscale/latest/installation/docker/

It first points to the images, then tells people to refer to the instructions of their own container platform or our Kubernetes instructions. And finally, we say
"Running Promscale directly using docker run is not recommended for production environments. This can be useful for testing purposes and is just provided as an example."